pennylane/tape/qnode.py

```diff
@@ -720,8 +755,34 @@ def circuit(a, w):
     >>> drawer(a=2.3, w=[1.2, 3.2, 0.7])
     0: ──H──╭C────────────────────────────╭C─────────╭┤ ⟨Z ⊗ Z⟩
     1: ─────╰RX(2.3)──Rot(1.2, 3.2, 0.7)──╰RX(-2.3)──╰┤ ⟨Z ⊗ Z⟩
+
+    Circuit drawing works with devices with custom wire labels:
+
+    .. code-block:: python3
+
+        dev = qml.device('default.qubit', wires=["a", -1, "q2"])
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.Hadamard(wires=-1)
+            qml.CNOT(wires=["a", "q2"])
+            qml.RX(0.2, wires="a")
+            return qml.expval(qml.PauliX(wires="q2"))
+
+    When printed, the wire order matches the order defined on the device:
+
+    >>> print(circuit.draw())
+      a: ─────╭C──RX(0.2)──┤
+     -1: ──H──│────────────┤
+     q2: ─────╰X───────────┤ ⟨X⟩
+
+    We can use the ``wire_order`` argument to change the wire order:
+
+    >>> print(circuit.draw(wire_order=["q2", "a", -1]))

```

Comment:
Make sure the examples here use the functional API instead.

Reply:
Oops, good catch! The folly of copy and pasting...